CONFIG.txt

```diff
@@ -13,13 +13,13 @@ genome_name:
 output_dir: "ploidetect_out"
 temp_dir:
 
-# ploidetect_github_version should be a branch or tag.  Overriden by ploidetect_local_clone.
-ploidetect_github_version: v1.0.0
+# ploidetect_ver should be a branch or tag.  Overriden by ploidetect_local_clone.
+ploidetect_ver: v1.0.0
 # Leave ploidetect_local_clone blank or 'None' to download from github
-ploidetect_local_clone: /gsc/pipelines/Ploidetect/{ploidetect_github_version}
+ploidetect_local_clone: /gsc/pipelines/Ploidetect/{ploidetect_ver}
 
 # are we using docker?
-use-docker: 1
+use_docker: 0
```

Comment:
Hi @lculibrk,

These two changes might break backwards compatibility for your existing configs.

ploidetect_ver - just seemed more clear in code displays and more consistent.

'use-docker' vs 'use_docker' - just because the command-line over-rides '--config use_docker=1', works while 'use-docker' creates an argparse error.

Reply:
I have local changes to commit that will obsolete the need for the use[-_]docker in the config.